cas/cas.h

```diff
@@ -406,7 +405,7 @@ struct m0_cas_op {
  */
 struct m0_cas_dtm0_log_payload  {
         /**  CAS op */
-	struct m0_cas_op cdg_cas_op;
+	struct           m0_cas_op cdg_cas_op;
```

Comment:
Why do we need so many whitespaces here? Regarding indentation I meant to check the indent for         _/**  CAS op */_ comment, it looks as moved too much to the right.